Mathlib/Data/Array/Defs.lean

```diff
@@ -4,6 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Arthur Paulino, Floris van Doorn
 -/
 
+import Std
```

Comment:
We're going to want to avoid `import Std` in mathlib for the same reason that  `import Mathlib` is a bad idea. It won't be a problem for a while, but mathlib files at least should stick to precise imports.

Reply:
For the moment it is intentional so that we avoid duplicate definitions in mathlib and std.